src/QMCHamiltonians/tests/test_force.cpp

```diff
@@ -425,4 +425,84 @@ TEST_CASE("Ion-ion Force", "[hamiltonian]")
     REQUIRE(elecForce.forces[2][i] == Approx(0.0));
   }
 }
+
+TEST_CASE("AC Force", "[hamiltonian]")
+{
+  ParticleSet ions;
+  ParticleSet elec;
+
+  ions.setName("ion");
+  ions.create(1);
+  ions.R[0][0] = 0.0;
+  ions.R[0][1] = 0.0;
+  ions.R[0][2] = 0.0;
+
+  elec.setName("elec");
+  elec.create(2);
+  elec.R[0][0] = 0.0;
+  elec.R[0][1] = 1.0;
+  elec.R[0][2] = 0.0;
+  elec.R[1][0] = 0.4;
+  elec.R[1][1] = 0.3;
+  elec.R[1][2] = 0.0;
+
+  SpeciesSet& tspecies = elec.getSpeciesSet();
+  int upIdx            = tspecies.addSpecies("u");
+  //int chargeIdx = tspecies.addAttribute("charge");
+  int massIdx                 = tspecies.addAttribute("mass");
+  int eChargeIdx              = tspecies.addAttribute("charge");
+  tspecies(eChargeIdx, upIdx) = -1.0;
+  tspecies(massIdx, upIdx)    = 1.0;
+
+
+  // The call to resetGroups is needed transfer the SpeciesSet
+  // settings to the ParticleSet
+  elec.resetGroups();
+
+  SpeciesSet& ion_species           = ions.getSpeciesSet();
+  int pIdx                          = ion_species.addSpecies("H");
+  int pChargeIdx                    = ion_species.addAttribute("charge");
+  int pMembersizeIdx                = ion_species.addAttribute("membersize");
+  ion_species(pChargeIdx, pIdx)     = 1;
+  ion_species(pMembersizeIdx, pIdx) = 1;
+
+  ions.resetGroups();
+  // Must update ions first in SoA so ions.coordinates_ is valid
+  ions.update();
+
+  elec.addTable(ions);
+  elec.update();
+
+  // defaults
+  TrialWaveFunction psi;
+  QMCHamiltonian qmcHamiltonian;
+
+  ACForce force(ions, elec, psi, qmcHamiltonian);
```

Comment:
naming is going to become confusing once you start checking results i.e. the `Force` type shows up in tests.

Reply:
@PDoakORNL since we are switching `Force_t` to `Forces`, do you think the naming can stay consistent with existing tests coming from other force types? At least for now, naming refactoring would have a larger scope in a separate PR.

Reply:
I think that would be nice ultimately but I don't like to see "new" code going in with old names, my experience is then there is no pressure to update at all. If this just makes the naming inconsistent fine,  however if breaks all those tests and will bring in many more files into this PR maybe it should get split into a separate PR up to you.

In the case of other force tests I'm not even sure if those will be kept once @rcclay 's new implementation is in so I wouldn't worry about them unless it breaks the build.